sdk/core/core-client/review/core-client.api.md

```diff
@@ -21,7 +21,7 @@ import { TransferProgressEvent } from '@azure/core-rest-pipeline';
 // @public
 export function authorizeRequestOnClaimChallenge(onChallengeOptions: AuthorizeRequestOnChallengeOptions): Promise<boolean>;
 
-// @public
+// @public (undocumented)
```

Comment:
@bzhang0 Looks that you will need to merge the latest from upstream/main as these documentations are added recently.